### ISSUE TITLE
Subtitles are not fitting their usual format

### ISSUE THREAD

**Issue by Milo-Koz** [crunchy-labs/crunchy-cli#421]
**Describe the bug**
When I download an episode with the crunchy-cli command, the subtitles are in a weird format, not matching crunchyroll's.

**To Reproduce**
Steps / command to reproduce the behavior:

1. Download an archive
2. Read the file with a video player (I'm using VLC)

**Expected behavior**
Subtitles should look like this : 
<img width="1440" alt="Capture d’écran 2024-05-17 à 16 15 43" src="https://github.com/crunchy-labs/crunchy-cli/assets/79766469/57f3c54a-fa38-4cea-8f9e-60c560627847">

**Screenshots**
But instead it looks like this :
<img width="1440" alt="Capture d’écran 2024-05-17 à 15 57 38" src="https://github.com/crunchy-labs/crunchy-cli/assets/79766469/c5699913-0589-45ea-83a1-7820825fad5b">

**Client (please complete the following information):**
 - OS: Mac Ventura (M1, 2020) + Windows 10
 - Version : v3.6.2

**Comment by bytedream** [comment]
What was the exact command you were using?

**Comment by Milo-Koz** [comment]
I used this :
```
crunchy-cli archive -a ja-JP -s fr-FR --default-subtitle fr-FR -o "E{episode_number} - {title}.mkv" https://www.crunchyroll.com/fr/watch/GRQ49EP1Y/come-back-robin-showdown-with-cp9
```
Also tried with "--include-fonts" argument but it was the same result

**Comment by Milo-Koz** [comment]
Works like a charm now, thanks !

**Comment by bytedream** [comment]
The fix is now also available in [v3.6.3](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.6.3)
